.landscaper/logging-stack/resources-cert-manager.yaml

```diff
@@ -12,53 +12,53 @@ input:
 type: helm.io/chart
 name: cert-manager-chart
 relation: external
-version: v1.13.0
+version: v1.13.1
 access:
   type: ociRegistry
-  imageReference: eu.gcr.io/gardener-project/landscaper-service/charts/cert-manager@sha256:ead441a652e94abedf699becf71ab24616f5464efde212c388b05d20ea738b3a
+  imageReference: eu.gcr.io/gardener-project/landscaper-service/charts/cert-manager@sha256:674552b27878738099d7d6e420257470fd16495e2698af2596eb785f32f52166
```

Comment:
There is another image in the registry with tag `v1.13.1` and digest `sha256:8587822f6794e732e9b65f30eaa462079916eb7a31bef93d72cf8ce2fd4e3175`. Is this the right one?